rpi-gpio.js

```diff
@@ -181,7 +181,14 @@ function Gpio() {
                 exportPin(pinForSetup, next);
             },
             function(next) {
-                setEdge(pinForSetup, edge, next);
+              async.retry({times: 10, interval: 100},
+                function(cb){
+                  setEdge(pinForSetup, edge, cb);
+                },
+                function(err){
+                  // wrapped here because waterfall can't handle positive result
```

Comment:
Not sure what this comment means by positive result... AFAICT, this final callback is required to chain into the waterfall always? If we omitted it, we'd always just deadlock.